settings.gradle

```diff
@@ -61,6 +61,9 @@ include ':agent:agent-tooling'
 include ':agent:agent-for-testing'
 include ':agent:instrumentation:applicationinsights-web-2.3'
 include ':agent:instrumentation:azure-functions'
+include ':agent:instrumentation:netty-4.0'
```

Comment:
should we drop the support for netty-4.0? i thought i posted a comment.. somehow got removed. i can the statsbeat data with you offline.. very little usage.

Reply:
I replied in Teams 😄